docker-compose.yaml

```diff
@@ -0,0 +1,53 @@
+version: '3.1'
```

Comment:
Any reason for using version 3.1 here? Latest version of docker compose is 3.7 which supports Docker Engine 18.06.0 and up. I believe the latest version of Docker Engine is 19.03 atm.

Reply:
No specific reason. I used the default `docker-compose.yaml` from the [official WordPress docker image documentation](https://github.com/docker-library/docs/blob/master/wordpress/README.md#-via-docker-stack-deploy-or-docker-compose) to start working on the setup and didn't think to change the version. I updated it to 3.7 now. Thanks for catching it!